tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/country_language_population.tsv

```diff
@@ -787,10 +787,13 @@ Luxembourg	LU	"605,764"	100%	"62,110,000,000"	official	French	fr	87%
 Luxembourg	LU	"605,764"	100%	"62,110,000,000"	official	German	de	63%
 Luxembourg	LU	"605,764"	100%	"62,110,000,000"	official	Luxembourgish	lb	67%	5%		"http://www.ethnologue.com/show_language.asp?code=ltz Some 99% of users are literate in French or German. For languages not customarily written, the writing population is artificially set to 5% in the absence of better information."
 Luxembourg	LU	"605,764"	100%	"62,110,000,000"		Portuguese	pt	16%			https://en.wikipedia.org/wiki/Portuguese_Luxembourger
-Macao SAR China	MO	"606,340"	96%	"71,820,000,000"		Chinese	zh	5%			Hans literacy is unknown; set to 5% artificially pending better or official figures.
-Macao SAR China	MO	"606,340"	96%	"71,820,000,000"	official	Chinese (Traditional)	zh_Hant	98%
-Macao SAR China	MO	"606,340"	96%	"71,820,000,000"		English	en	"13,900"			https://www.cia.gov/library/publications/the-world-factbook/geos/mc.html and https://en.wikipedia.org/wiki/Macau
-Macao SAR China	MO	"606,340"	96%	"71,820,000,000"	official	Portuguese	pt	5%			http://en.wikipedia.org/wiki/Geographic_distribution_of_Portuguese Macao reported 5% native Portuguese speakers.
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"		Chinese	zh	5%			Hans literacy is unknown; set to 5% artificially pending better or official figures.
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"	official	Chinese (Traditional)	zh_Hant	98%
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"		English	en	22.7%			https://www.dsec.gov.mo/getAttachment/6cb29f2f-524a-488f-aed3-4d7207bb109e/E_CEN_PUB_2021_Y.aspx 2021 Census, counting people who are fluent in the language
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"	official	Portuguese	pt	2.3%			https://www.dsec.gov.mo/getAttachment/6cb29f2f-524a-488f-aed3-4d7207bb109e/E_CEN_PUB_2021_Y.aspx 2021 Census, counting people who are fluent in the language
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"	de_facto_official	Cantonese	yue	86.2%			https://www.dsec.gov.mo/getAttachment/6cb29f2f-524a-488f-aed3-4d7207bb109e/E_CEN_PUB_2021_Y.aspx 2021 Census, counting people who are fluent in the language
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"		Filipino	fil	"20,879"			https://www.dsec.gov.mo/getAttachment/6cb29f2f-524a-488f-aed3-4d7207bb109e/E_CEN_PUB_2021_Y.aspx 2021 Census, counting people who are fluent in the language
+Macao SAR China	MO	"682,070"	96%	"71,820,000,000"		Hokkien	nan	3.7%			https://www.dsec.gov.mo/getAttachment/7a3b17c2-22cc-4197-9bd5-ccc6eec388a2/E_CEN_PUB_2011_Y.aspx 2011 Census -- the language is not distinguished in the 2021 census
```

Comment:
is this the reference intended?

Reply:
Yes -- do you want me to re-write it?

Hokkien is a distinguished language in the 2011 Census but it does not appear in the 2021 Census for Macao.